### PR TITLE
Require Drupal Core 8.9

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Updated drupal core and several modules.
- Requires https://github.com/SU-SWS/stanford_image_styles/pull/54

# Need Review By (Date)
- asap

# Urgency
- high

# Steps to Test
1. review the tests

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -140,7 +139,7 @@
         "su-sws/stanford_date_formats": "dev-8.x-1.x",
         "su-sws/stanford_events": "dev-8.x-1.x",
         "su-sws/stanford_fields": "dev-8.x-1.x",
-        "su-sws/stanford_image_styles": "dev-8.x-1.x",
+        "su-sws/stanford_image_styles": "dev-pookmish-patch-1",
```
Comment (sherakama) [inline composer.json:142]:
```suggestion
        "su-sws/stanford_image_styles": "dev-8.x-1.x",
```

File: composer.json [hunk 2]
```diff
@@ -102,10 +102,9 @@
         "drupal/field_group": "^3.0@rc",
         "drupal/field_permissions": "^1.0@RC",
         "drupal/field_validation": "^1.0-alpha",
-        "drupal/file_mdm": "~1.1",
```
Comment (sherakama) [inline composer.json:105]: Where did this go?

Comment (pookmish) [reply composer.json:105]: its requirement of imagamagick module. so its inherited

Comment (pookmish) [reply composer.json:105]: `2.0` version is

File: config/sync/core.extension.yml [hunk 3]
```diff
@@ -115,6 +115,7 @@ module:
   shs: 0
   simplesamlphp_auth: 0
   smart_date: 0
+  sophron: 0
```
Comment (sherakama) [inline config/sync/core.extension.yml:118]: Where is the being included?

Comment (sherakama) [reply config/sync/core.extension.yml:118]: Why do we needs it?

Comment (pookmish) [reply config/sync/core.extension.yml:118]: the latest imagemagick requires it https://git.drupalcode.org/project/imagemagick/-/blob/8.x-3.x/imagemagick.info.yml#L10

File: composer.json (hunk 2 above)
Comment (sherakama) [reply composer.json:105]: ah: https://git.drupalcode.org/project/imagemagick/-/blob/8.x-3.x/imagemagick.info.yml#L10

Comment (sherakama) [reply composer.json:105]: Thank you

**Comment by pookmish** [comment]
hooray

**Pull request merged**